app/src/main/java/org/mozilla/fenix/HomeActivity.kt

```diff
@@ -220,7 +220,7 @@ open class HomeActivity : LocaleAwareAppCompatActivity(), NavHostActivity {
             shouldNavigateBrowserFragmentOnCouldStart(savedInstanceState)
         ) {
             navigateToBrowserOnColdStart()
-        } else {
+        } else if (FeatureFlags.showStartOnHomeSettings) {
```

Comment:
Can we use `shouldStartOnHome()` here instead of this?

Reply:
Good point, we can but I would like to make explicit that we have to remove this check when removing the feature flag. As internally `shouldStartOnHome()` does a read from disk, that was already performed in the if condition statement above.

Reply:
That makes sense 👍